src/store.js

```diff
@@ -1,6 +1,202 @@
+import Vue from "vue";
+import { fetchBreeds } from "./data.js";
+
+const state = Vue.observable({
+  breedsList: [],
+  searchList: [],
+  isLoading: false,
+  error: "",
+  searchQuery: "",
+  message: "",
+  isInputDisabled: false,
+});
+const getters = {

```

Comment:
Sem poizkušal narediti kakor si omenil, torej da v storu nimam recimo ločenih state, actions, getters ampak je store eno funkcija ki vrne actione in getterje...ampak mi ni delalo...nisem mogu dostopat do state-date
Tako sem imel narejeno
`export default Vue.observable({
  state: {
    breedsList: [],
    searchList: [],
    isLoading: false,
    error: "",
    searchQuery: "",
    message: "",
    isInputDisabled: false,
  },
  getters: {
    displayedList() {
      if (state.searchQuery) {
        return state.searchList;
      } else {
        console.log(state.breedsList); // Access state properties using state
        return state.breedsList;
      }
    },
    // Other getters...
  },
  actions: {
    // ...
  },
});`

Maš idejo, zakaj ne moram dostopati?
 `Vue.observable` sem pa uporabil kar omogoča reactivity, drugače se mi data sploh ni naložila

Reply:
kako si pa pol hotu uporabiti to v komponenti?

Reply:
ne gre pol več tko.. ti v store file exportaš funkcijo k ti da state, actions, getters.. tko da je treba temu primerno tud uporabit.. še mal vaje ;)

Reply:
samo poglej kaj imaš, kaj dobiš in kako to uporabit, ni nč posebnega..